portfolio/src/main/webapp/script.js

```diff
@@ -64,7 +61,7 @@ function parseSomething(){
         numberCommentsDisplay = Math.min(numberCommentsDisplay, object.length);
 
         for (var i = 0; i < numberCommentsDisplay; i++) {
-            commentsListElement.appendChild(createListElement(sortedObjs[i]));
+            commentsListElement.appendChild(createListElement(object[i]));
```

Comment:
This seems like another bug from the previous PR.

Reply:
The PR description is updated! :)